src/models/spaces/breeding/breeding.ts

```diff
@@ -4,20 +4,26 @@ import { BackpackMouse, BackpackMouseType } from "../../backpack-mouse";
 import { breed, createGamete, fertilize } from "../../../utilities/genetics";
 import { ToolbarButton } from "../populations/populations";
 import uuid = require("uuid");
-
+import { RightPanelType } from "../../../models/ui";
 export const BreedingTypeEnum = types.enumeration("type", ["litter", "singleGamete"]);
 export type BreedingType = typeof BreedingTypeEnum.Type;
 
 const BreedingInteractionModeEnum = types.enumeration("interaction", ["none", "breed", "select", "inspect"]);
 export type BreedingInteractionModeType = typeof BreedingInteractionModeEnum.Type;
 
+const BreedingChartTypeEnum = types.enumeration("chart", ["color", "genotype", "sex"]);
+export type BreedingChartType = typeof BreedingChartTypeEnum.Type;
+
 const NestPair = types.model({
   id: types.optional(types.identifier, () => uuid()),
   leftMouse: BackpackMouse,
   rightMouse: BackpackMouse,
   leftMouseBackpackId: types.maybe(types.string),
   rightMouseBackpackId: types.maybe(types.string),
-  label: types.string
+  label: types.string,
+  currentBreeding: false, // TODO: might be replaced based on breeding changes
+  activeBreeding: false, // TODO: might be replaced based on breeding changes
+  numOffspring: 0 // TODO: replace by requesting size of offspring data structure
```

Comment:
These new properties will probably be adjusted based on how we keep track of the following:
- whether or not: a nest pair has been selected for breeding
- whether or not: a nest pair is the current breeding pair (the pair shown in the left panel)
- how many offspring a pair has